### PR TITLE
Update packages, fix tab length

### PR DESCRIPTION
**Pull request by Henryws**

### CODE REVIEW

File: packages/minecraft-launcher/minecraft-launcher.pacscript [hunk 1]
```diff
@@ -8,8 +8,8 @@ description="Official Minecraft Launcher"
 hash="e84cadeb55db50d673bb97d3ad1c9ad52f4e5999ed25fdcf3859cec43b5a449e"
 
 prepare() {
-        wget -q https://gist.githubusercontent.com/aranym0/c752045e3b71bdb5d8c208f57dc12dba/raw/4aa522fe02cd827300ce102b3d5f38ba38798a9c/minecraft-launcher.desktop
-        wget -q https://launcher.mojang.com/download/minecraft-launcher.svg
+        sudo wget -q https://gist.githubusercontent.com/aranym0/c752045e3b71bdb5d8c208f57dc12dba/raw/4aa522fe02cd827300ce102b3d5f38ba38798a9c/minecraft-launcher.desktop
+        sudo wget -q https://launcher.mojang.com/download/minecraft-launcher.svg
```
Comment (wizard-28) [inline packages/minecraft-launcher/minecraft-launcher.pacscript:12]:
```suggestion
        wget -q https://gist.githubusercontent.com/aranym0/c752045e3b71bdb5d8c208f57dc12dba/raw/4aa522fe02cd827300ce102b3d5f38ba38798a9c/minecraft-launcher.desktop
        wget -q https://launcher.mojang.com/download/minecraft-launcher.svg
```
These `sudo` may not be required, as it's downloading stuff in `/tmp/pacstall`, which is accessible by user

File: packages/nemo/nemo.pacscript [hunk 2]
```diff
@@ -9,14 +9,14 @@ gives="nemo"
...
 prepare() {
-          meson --prefix="/usr/src/pacstall/nemo" build 
+    meson --prefix="/usr/src/pacstall/nemo" build 
 }
 
 build() {
-        cd build
-        sudo ninja -j$(nproc)
+    cd build
+    sudo ninja -j$(nproc)
```
Comment (wizard-28) [inline packages/nemo/nemo.pacscript:17]:
```suggestion
    ninja -j$(nproc)
```

Same here, it's doing stuff in `/tmp/pacstall`

File: packages/pacget/pacget.pacscript [hunk 3]
```diff
@@ -12,20 +12,17 @@ debian=TRUE
...
-true
+build() {
+    true
 }
 #this just coppies the files over and makes the file executable
-install() 
-{
+install() {
 	sudo cp -p pacget /usr/src/pacstall/pacget/usr/bin/pacget
 	sudo chmod 755 /usr/src/pacstall/pacget/usr/bin/pacget
```
Comment (wizard-28) [inline packages/pacget/pacget.pacscript:27]:
```suggestion
 	sudo install -Dm755 "pacget" "/usr/src/pacstall/pacget/usr/bin/pacget"
```

Compress the two lines into one

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Comment by Henryws** [comment]
Merge?

**Comment by wizard-28** [comment]
See the requested changes

**Pull request merged**